src/xcb.rs

```diff
@@ -59,33 +79,52 @@ impl AsRef<XCBConnection> for XCBConnection {
 
 impl Clone for XCBConnection {
     fn clone(&self) -> XCBConnection {
-        unsafe { from_glib_none(self.to_glib_none().0) }
+        unsafe { Self::from_raw_none(self.to_raw_none()) }
```

Comment:
I don't really know if it's better to remove the ```unsafe``` here or leave it there and mark the impl ```unsafe``` like #127 ?